docker-compose.yml

```diff
@@ -41,6 +41,7 @@ services:
       - IT_INVESTMENT_EMAIL=success@simulator.amazonses.com
       - EASI_HELP_EMAIL=success@simulator.amazonses.com
       - TRB_EMAIL=success@simulator.amazonses.com
+      - OIT_FEEDBACK_CHANNEL_SLACK_LINK=https://oddball.slack.com/archives/C059N01AYGM
```

Comment:
I don't _think_ you need to include the value here. If you look at some of the values in the lines further up, they just provide the env var name, which means it'll pull from the hosts envs (which ought to work if we're setting it everywhere we're supposed to)

Reply:
I removed it here [fd298b1](https://github.com/CMS-Enterprise/easi-app/pull/2859/commits/fd298b17ca3dad57ea91256b216d3942cc04ba69) and the e2e suite failed